webpack/assets/javascripts/react_app/routes/index.js

```diff
@@ -31,18 +32,29 @@ const AppSwitcher = () => {
   };
 
   return (
-    <Switch>
-      {routes.map(({ render: Component, path, ...routeProps }) => (
-        <Route
-          path={path}
-          key={path}
-          {...routeProps}
-          render={componentProps => handleRoute(Component, componentProps)}
-        />
-      ))}
-      <Route render={handleFallbackRoute} />
-    </Switch>
+    <React.Fragment>
+      <Switch>
+        {routes.map(({ render: Component, path, ...routeProps }) => (
+          <Route
+            path={path}
+            key={path}
+            {...routeProps}
+            render={componentProps => handleRoute(Component, componentProps)}
+          />
+        ))}
+        <Route render={handleFallbackRoute} />
+      </Switch>
+      {props.children}
```

Comment:
I'm not sure how this is related to the cancel button mounting? and why this affects only that component?

Reply:
Because for cancel button is wrapped with Router and AppSwitcher so it can be handled by react-router.